lib/puppet/face/epp.rb

```diff
@@ -153,7 +153,7 @@
       # Print to a buffer since the face needs to return the resulting string
       # and the face API is "all or nothing"
       #
-      buffer = StringIO.new
+      buffer = StringIO.new('', 'r+:UTF-8')
```

Comment:
`r+:UTF-8` is meaningless to `StringIO`.  Instead, ensure that `''` is in `UTF-8`